scripts/build-llm-docs.ts

```diff
@@ -0,0 +1,115 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import { glob } from 'node:fs/promises'
+
+const frontmatterRegex = /^\n*---(\n.+)*?\n---\n/
+
+const docsDir = path.resolve('docs')
+
+const sliceExt = (file: string) => {
+  return file.split('.').slice(0, -1).join('.')
+}
+
+const extractLabel = (file: string) => {
+  return sliceExt(file.split('/').pop() || '')
+}
+
+function capitalizeDelimiter(str) {
+  return str
+    .split('-')
+    .map((s) => s.charAt(0).toUpperCase() + s.slice(1))
+    .join('-')
+}
+
+async function generateLLMDocs() {
+  const outputListFile = path.resolve('public/llms.txt')
+
+  const optionalFiles = await glob('**/*.md', { cwd: docsDir })
+
+  const optionals: string[] = []
+
+  for await (const file of optionalFiles) {
+    optionals.push(
+      `- [${capitalizeDelimiter(extractLabel(file)).replace(/-/, ' ')}](https://hono.dev/docs/${sliceExt(file)})`
+    )
+  }
+
+  fs.writeFileSync(
+    outputListFile,
+    [
+      '# Hono',
+      '',
+      '> Hono - means flame🔥 in Japanese - is a small, simple, and ultrafast web framework built on Web Standards. It works on any JavaScript runtime: Cloudflare Workers, Fastly Compute, Deno, Bun, Vercel, Netlify, AWS Lambda, Lambda@Edge, and Node.js.',
+      '',
+      'Important notes:',
```

Comment:
Are these notes necessary? These are not described on the website. I think the content should be the same as the one on the website.